app/services/sufia/statistics/collections.rb

```diff
@@ -0,0 +1,31 @@
+module Sufia
+  module Statistics
+    class Collections
+      # @param [Fixnum] delta_x change in x (in days)
+      # @param [Date] x_min minimum date
+      # @param [Date] x_max max date
+      def initialize(delta_x: 7, x_min: 1.month.ago.beginning_of_day, x_max: Date.today.end_of_day)
```

Comment:
I see this is a pattern from the other stats classes, and there's probably value in keeping it. I'm surprised it isn't a `Range` type :shrug:.